python/tests/io_tests/test_paths.py

```diff
@@ -0,0 +1,41 @@
+import os
+from nose2.tools import params
+from unittest import TestCase
+
+from aac.io.paths import sanitize_filesystem_path
+
+WORKING_TEST_DIR = os.getcwd()
+
+
+def _get_working_directory_parent_directory(level_of_directories_up: int = 1) -> str:
```

Comment:
Is there a reason not to make this a general utility?

Reply:
I elected not to since it felt like a very special use case/need.